src/index.js

```diff
@@ -0,0 +1,139 @@
+    // } else if (inputSky === "snowy") {
+    //     gardenGifs = "snowy";
+    // }
+    
+    
+    const gardenContent = document.querySelector(".gardenContainer");
+    gardenContent.className = "gardenContainer"
+    gardenContent.classList.add(inputSky);
+}
+const changeCityName = () => {
+    
+    const cityNameInputEl= document.getElementById("cityNameInput").value; 
+    const headerCityEl = document.getElementById("CityName"); 
+    headerCityEl.textContent = cityNameInputEl;
+    console.log("citygirls")
+}
+const cityReset = () => {
+
+    const cityNameInput = document.getElementsById("cityNameInput");
+    cityNameInput.value = "Miami"; 
+    cityNameInput.value.classList.add(cityNameInput);
+    console.log("inthecitygirls")
+    changeCityName();
+    
+
+}; 
+
+const changeTempColor = () => {
+
+    const tempColor = document.getElementById("tempvalue");
+        let color = "blue"; 
+        if (state.tempchange >=80){
+            color = "rgba(24, 17, 24, 0.836)";
+        console.log("hotttttttttt")
+        }else if (state.tempchange  >=70){
+            color = "rgb(117, 87, 117)";
+        }else if (state.tempchange  >=60){
+            color = "rgb(175, 124, 175)";
+        }else if (state.tempchange  >=50){
+            color = "black";
+        }
+        tempColor.classList = color; 
+
+
+}
+
+const changeGarden = ()=> {
+
+    const landContainer = document.querySelector("#landscape")
+    let landscape = "🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶"; 
+    if (state.tempchange>=80){
+        landscape = "🥵🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🥵"; 
+    } else if (state.tempchange >=70){
+        landscape = "✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️"; 
+    }else if (state.tempchange >=60){
+        landscape = "🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂"; 
+    }
+
+    landContainer.textContent = landscape
+
+
+}
+const changeTemp = tempvalue => {
+
+    const changeTempContainer = document.getElementById('tempvalue');
+    changeTempContainer.textContent = tempvalue; 
+    tempGardenvlue(tempvalue);
+    changeGarden(tempvalue);
+
+
+
+}
+
+
+
+    //register the button, telling the website to listen for a click 
+const registerEventHandlers = () =>{
+    const increseButtonElement = document.getElementById('increaseButton');
+    increseButtonElement.addEventListener("click", clickButtonUp)
+    console.log("registerEventHandlers")
+
+    const decreaseButtonElement = document.getElementById('decreaseButton');
+    decreaseButtonElement.addEventListener("click", clickButtonDown)
+    console.log("registerTheseHands")
+
+    const inputSky = document.querySelector(".skyDropMenu");
+    inputSky.addEventListener("change",changeSky)
+
+    changeCityName();
+    const cityNameInputEl= document.getElementById("cityNameInput"); 
+    cityNameInputEl.addEventListener("input", changeCityName)
+
+    const resetButton = document.getElementById("cityNameInput")
+    resetButton.addEventListener("click", changeCityName)
```

Comment:
instead of calling the changeCityName function, you should create a function to reset the city name back to the default city name